webpack.libraries.config.js

```diff
@@ -42,7 +42,8 @@ module.exports = (env, argv) => {
         output: {
             filename: 'jahia-commons.dll.js',
             path: path.resolve(__dirname, 'src/main/resources/javascript/commons/'),
-            library: 'jahiaCommons'
+            library: 'jahiaCommons',
+            publicPath: '/modules/app-shell/javascript/commons/'
```

Comment:
this may actually not work if jahia is not installed in / . the publicPath is already initialized dynamically here : https://github.com/Jahia/app-shell/blob/0fea3ce8a4bfe363eea8c9a036829b6d8bee2f53/src/javascript/jahia.js#L3  , this should actually be used to resolve fonts too ?

Reply:
It doesn't seem to resolve the font right now. According to this answer, it's because of the relative url specified in the css. It seems like another option could be to extract the css from the js bundle - do you see any issues with that?

Reply:
It was actually the issue solved by @FBerthelot for loading moonstone fonts : BACKLOG-12748 . I don't know why it's not working the same way for design-systemkit